models/user/user.go

```diff
@@ -537,6 +537,7 @@ var (
 		"gitea-actions",
 	}
 
+	// DON'T ADD ANY NEW STUFF, WE SOLVE THIS WITH `/user/{obj}`  PATHS!
 	reservedUserPatterns = []string{"*.keys", "*.gpg", "*.rss", "*.atom"}
```

Comment:
Maybe remove these later then if it can be fully solved via router?

Reply:
droping already reserved stuff would be another pull ... and we already have reserved it so it's fine just dont add new stuff

Reply:
PS: in this regards, we do mimic github witch does the same!

e.g.: https://github.com/silverwind.keys

Reply:
If `alicia.keys.keys` works, and the router bug is the only reason we reserve, let's drop the reservations.